webpack.config.js

```diff
@@ -85,13 +86,13 @@ module.exports = {
     quiet: false,
     disableHostCheck: true,
     host: '0.0.0.0',
-    port: 3000,
+    port: new URL(frontendUrl).port,
```

Comment:
If you were to assign `new URL(frontendUrl)` to a constant on line 11, then you'll avoid duplicate instantiation of a `new URL` construct.  Then just use the const on lines 89 and 95.

Reply:
Neater this way for sure. Resolved